src/Symfony/Component/Routing/Loader/AttributeClassLoader.php

```diff
@@ -93,7 +93,7 @@ public function load(mixed $class, ?string $type = null): RouteCollection
         foreach ($class->getMethods() as $method) {
             $this->defaultRouteIndex = 0;
             $routeNamesBefore = array_keys($collection->all());
-            foreach ($this->getAnnotations($method) as $annot) {
+            foreach ($this->getAttributes($method) as $annot) {
```

Comment:
maybe renamed also all $annot by $attr?

Reply:
also in src/Symfony/Bundle/FrameworkBundle/Routing/AttributeRouteControllerLoader.php